core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -218,6 +221,128 @@
           Arrays.asList(RPC_SERVER_METHOD));
 
+  // Interval Stats
+  static final Duration MINUTE = Duration.create(60, 0);
+  static final Duration HOUR = Duration.create(60 * 60, 0);
+
+  /**
+   * Census defined rpc client {@link IntervalViewDescriptor}s.
+   */
+  public static final IntervalViewDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client roundtrip latency",
+          "Latency in msecs",
+          RPC_CLIENT_ROUNDTRIP_LATENCY,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_REQUEST_BYTES_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client request_bytes",
+          "Request MB",
+          RPC_CLIENT_REQUEST_BYTES,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_RESPONSE_BYTES_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client response_bytes",
+          "Response MB",
+          RPC_CLIENT_RESPONSE_BYTES,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_ERROR_COUNT_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client error count",
+          "RPC Errors",
+          RPC_CLIENT_ERROR_COUNT,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client uncompressed_request_bytes",
+          "Uncompressed Request MB",
+          RPC_CLIENT_UNCOMPRESSED_REQUEST_BYTES,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_UNCOMPRESSED_RESPONSE_BYTES_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client uncompressed_response_bytes",
+          "Uncompressed Response MB",
+          RPC_CLIENT_RESPONSE_BYTES,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_SERVER_ELAPSED_TIME_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client server_elapsed_time",
+          "Server elapsed time in msecs",
+          RPC_CLIENT_SERVER_ELAPSED_TIME,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  /**
+   * Census defined rpc server {@link IntervalViewDescriptor}s.
+   */
```

Comment:
My comments in the RPC client section apply here, too, except for the comment about units.